### PR TITLE
more stuff

### PR DESCRIPTION
**Pull request by JosuaKrause**
- add make local-build command
- work around missing file
- replace call of async function with a delayed approach
- only start mosaic animation after image has fully loaded
- make the service worker install itself by skipping the wait

### CODE REVIEW

File: routes/upload/index.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ const { BlobServiceClient } = require('@azure/storage-blob')
 const fs = require('fs')
 const { join } = require('path')
 
-const { createContainer } = require('./container_functions.js')
+// TO DO: add the file to the repo and uncomment the line below
```
Comment (JosuaKrause) [inline routes/upload/index.js:7]: please do that before mergin @myjyby

Comment (JosuaKrause) [reply routes/upload/index.js:7]: at all 4 places

File: routes/request/img/buffer.js [hunk 2]
```diff
@@ -2,7 +2,8 @@ const { app_storage } = include('config/')
 const fs = require('fs')
 const path = require('path')
 
-const { createContainer } = include('routes/upload/container_functions.js');
+// TO DO: add the file to the repo and uncomment the line below
```
Comment (JosuaKrause) [inline routes/request/img/buffer.js:5]:
```suggestion
```

File: routes/request/img/buffer.js [hunk 3]
```diff
@@ -2,7 +2,8 @@ const { app_storage } = include('config/')
...
+// const { createContainer } = include('routes/upload/container_functions.js');
```
Comment (JosuaKrause) [inline routes/request/img/buffer.js:6]:
```suggestion
const { createContainer } = include('routes/upload/container_functions.js');
```

File: routes/request/img/url.js [hunk 4]
```diff
@@ -4,7 +4,8 @@ const path = require('path');
 const fetch = require('node-fetch');
 const { v4: uuidv4 } = require('uuid');
 
-const { createContainer } = include('routes/upload/container_functions.js');
+// TO DO: add the file to the repo and uncomment the line below
```
Comment (JosuaKrause) [inline routes/request/img/url.js:7]:
```suggestion
```

File: routes/request/img/url.js [hunk 5]
```diff
@@ -4,7 +4,8 @@ const path = require('path');
...
+// const { createContainer } = include('routes/upload/container_functions.js');
```
Comment (JosuaKrause) [inline routes/request/img/url.js:8]:
```suggestion
const { createContainer } = include('routes/upload/container_functions.js');
```

File: routes/save/img/index.js [hunk 6]
```diff
@@ -1,5 +1,6 @@
 const { modules, app_storage } = include('config/');
-const { createContainer, moveBlob } = include('routes/upload/container_functions.js');
+// TO DO: add the file to the repo and uncomment the line below
+// const { createContainer, moveBlob } = include('routes/upload/container_functions.js');
```
Comment (JosuaKrause) [inline routes/save/img/index.js:3]:
```suggestion
const { createContainer, moveBlob } = include('routes/upload/container_functions.js');
```

File: routes/upload/index.js (hunk 1 above)
Comment (JosuaKrause) [reply routes/upload/index.js:7]:
```suggestion
```

File: routes/upload/index.js [hunk 7]
```diff
@@ -4,7 +4,8 @@ const { BlobServiceClient } = require('@azure/storage-blob')
...
+// const { createContainer } = require('./container_functions.js')
```
Comment (JosuaKrause) [inline routes/upload/index.js:8]:
```suggestion
const { createContainer } = require('./container_functions.js')
```

**Review by myjyby — APPROVED** [review]
lgtm

**Pull request merged**